docs/faq.md

```diff
@@ -0,0 +1,21 @@
+# Frequently Asked Questions (FAQ)
+
+## 1. How to use the demo?
+
+Simply enter the following link to demo and click [Link to Demo](https://google.aurelius-atlas.com/demo/).
```

Comment:
This is not the same demo link we use elsewhere.

Further, could you please use the "Go to demo" button that we also have on the main page? This adds some visual consistency.